### ISSUE TITLE
go 1.19.2  build error on mac  and  detail error is : go-openai@v1.26.0/batch.go:205:16:   undefined: errors.Join

### ISSUE THREAD

**Issue by hachi-zzq** [sashabaranov/go-openai#774]
Your issue may already be reported!
Please search on the [issue tracker](https://github.com/sashabaranov/go-openai/issues) before creating one.

**Describe the bug**

when my golang version is go1.19.2，i run the example code about CreateChatCompletion, report this error

```
../../../go/pkg/mod/github.com/sashabaranov/go-openai@v1.26.0/batch.go:205:16: undefined: errors.Join
```

**To Reproduce**
my example code is 

```
package main

import (
	"context"
	"fmt"
	openai "github.com/sashabaranov/go-openai"
)

func main() {
	client := openai.NewClient("sk-xxxE")
	resp, err := client.CreateChatCompletion(
		context.Background(),
		openai.ChatCompletionRequest{
			Model: openai.GPT4o,
			Messages: []openai.ChatCompletionMessage{
				{
					Role:    openai.ChatMessageRoleUser,
					Content: "hello!",
				},
			},
		},
	)

	if err != nil {
		fmt.Printf("ChatCompletion error: %v\n", err)
		return
	}

	fmt.Println(resp.Choices[0].Message.Content)
}

```
and the result is 
<img width="907" alt="image" src="https://github.com/sashabaranov/go-openai/assets/5241713/55f76fca-bc60-4afa-9f9f-8c0417a798aa">

my project is run on mac apple kernal , the detail is 
Apple M2 Pro
Mac OS Ventura 13.6.1 

my go version is
go version go1.19.2 darwin/arm64

**Screenshots/Logs**
<img width="918" alt="image" src="https://github.com/sashabaranov/go-openai/assets/5241713/c2f38848-73bd-46dc-9a62-c07b49bf34f8">
<img width="658" alt="image" src="https://github.com/sashabaranov/go-openai/assets/5241713/8601d428-7efd-4744-9114-46583c700f10">

**Environment (please complete the following information):**
 - go-openai version: v1.26.0
 - Go version: go1.19.2 darwin/arm64
 - OpenAI API version: v1
 - OS: Mac OS Ventura 13.6.1 

**Additional context**
there is no error when i run it on 1.18 or 1.20

**Comment by sashabaranov** [comment]
@hachi-zzq fixed in v1.26.1 https://github.com/sashabaranov/go-openai/releases/tag/v1.26.1

**Comment by hachi-zzq** [comment]
@sashabaranov thanks
